content/en-US/api/Serial.md

```diff
@@ -0,0 +1,86 @@
+---
+recipe: api-interface
+title: 'Serial'
+mdn_url: /en-US/docs/Web/API/Serial
+specifications: https://wicg.github.io/serial/#serial-interface
+browser_compatibility: api.Serial
+---
+
+**When this feature ships, the content below will live on MDN under
+[developer.mozilla.org/en-US/docs/Web/API](https://developer.mozilla.org/en-US/docs/Web/API).**
+
+## Description
+
+The `Serial` interface of the Web Serial API provides attributes and methods for
+finding and connecting to serial ports from a web page.
+
+This interface requires a secure context.
+
+## Constructor
+
+None.
+
+## Events
+
+**`Serial.onconnect`**
```

Comment:
What event interfaces are passed to the event callbacks? Please add that information all the way through so that when these pages go on MDN we can link to them.

Reply:
As with the Idle Detection API docs, how would you suggest I phrase this? These sections already use "called" instead of "fired" to refer to running event handlers. Would you suggest "called with an `Event`"?